vault/policy_store_test.go

```diff
@@ -353,3 +354,86 @@ func TestPolicyStore_PoliciesByNamespaces(t *testing.T) {
 		t.Fatalf("expected: %v\ngot: %v", expectedResult, out)
 	}
 }
+
+// TestPolicyStore_GetNonEGPPolicyType has five test cases:
+//   - happy-acl and happy-rgp: we store a policy in the policy type map and
+//     then look up its type successfully.
+//   - not-in-map-acl and not-in-map-rgp: ensure that GetNonEGPPolicyType fails
+//     returning a nil and an error when the policy doesn't exist in the map.
+//   - unknown-policy-type: ensures that GetNonEGPPolicyType fails returning a nil
+//     and an error when the policy type in the type map is a value that
+//     does not map to a PolicyType.
+func TestPolicyStore_GetNonEGPPolicyType(t *testing.T) {
```

Comment:
Well spotted, thanks! In this case I will be making sure it's the same in ENT, since the previous name of the test wasn't changed after the func it tests was renamed 😄